sql-plugin/src/main/scala/com/nvidia/spark/rapids/spill/SpillFramework.scala

```diff
@@ -1722,6 +1740,14 @@ object SpillFramework extends Logging {
     logInfo(s"Initialized SpillFramework. Host spill store max size is: $hostSpillStorageSizeStr.")
   }
 
+  def getHostStoreSpillableSummary: String = {
+    stores.hostStore.spillableSummary()
```

Comment:
If the stores are not initialized, or we are shutting down, this function could throw an `IllegalStateException`. Lets just try/catch that exception and say something like "Could not log spill summary, spill framework not initialized".

Reply:
fixed. please check again @abellina